### ISSUE TITLE
Getting "Page not found" when trying to run a php script

### ISSUE THREAD

**Issue by gkurl** [aspendigital/docker-octobercms#26]
Getting the following when trying to run a simple php script that sends a mail - even just a php script that echo's "hello" does not seem to work:

<img width="1051" alt="image" src="https://user-images.githubusercontent.com/34505859/68433557-b886ca00-01ae-11ea-9a4a-e07d7f8806ec.png">

But if I place say a text file in that exact same directory you see in the URL and try and access it via the web browser I am able to download/view the file.

What could be wrong here?

**Comment by petehalverson** [comment]
October CMS [blocks all PHP files, except index by default via .htaccess](https://github.com/octobercms/october/blob/master/.htaccess#L46-L51).

**Comment by gkurl** [comment]
Thanks for pointing me to that. Never knew. I have edited out and commented out that part in the .htaccess to test it out but am still seeing the page not found error...I have restarted the container multiple times so apache should be reset everytime...are there further steps involved specific to your image? I am using the :latest image. I also tried pointing it to a specific path and even tried specifying the .php file name.

Any pointers?
